packages/calcite-components/src/components/list/list.tsx

```diff
@@ -553,6 +557,16 @@ export class List
               <slot onSlotchange={this.handleDefaultSlotChange} />
             </tbody>
           </table>
+          <div
+            aria-live="polite"
+            class={CSS.noResultsContainer}
```

Comment:
Is this class mainly used for testing purposes? If so, maybe we can use a test data attribute (`data-test-id="no-results-container"` would be compatible with [this proposed test pattern](https://github.com/Esri/calcite-design-system/pull/8171)).